src/main/resources/templates/layout.html

```diff
@@ -48,6 +48,8 @@
       </div>
     </div>
     <nav th:unless="${hideNav}" class="text-right">
+      <a th:href="@{/runs}" class="text-green-200 font-bold hover:underline">Runs</a>
+      <span class="text-green-200 font-bold"> | </span>
```

Comment:
This being green makes it blend in with the links. Maybe use the same grey as the line above it?